graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/views/ElasticsearchBackend.java

```diff
@@ -255,7 +254,7 @@ public QueryResult doRun(SearchJob job, Query query, ESGeneratedQueryContext que
                     return new SearchRequest()
                             .source(searchTypeQueries.get(searchTypeId))
                             .indices(indices.toArray(new String[0]))
-                            .indicesOptions(IndicesOptions.fromOptions(false, false, true, false));
+                            .indicesOptions(IndicesOptions.fromOptions(true, true, true, false));
```

Comment:
The same as here: https://github.com/Graylog2/graylog2-server/pull/18146/files#r1479887434

Reply:
done